pkg/kube/kubehttp.go

```diff
@@ -168,7 +170,7 @@ func (c *KubeHTTP) SecretPresent(ctx context.Context, secretName string, namespa
 func (c *KubeHTTP) GetApplications(ctx context.Context, namespace string) ([]wego.Application, error) {
 	result := wego.ApplicationList{}
 
-	if err := c.Client.List(ctx, &result); err != nil {
+	if err := c.Client.List(ctx, &result, namespaceOpts(namespace)); err != nil {
```

Comment:
I think you could simplify this to https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/client#InNamespace and remove the `namespaceOpts` function below.

```go
if err := c.Client.List(ctx, &result, client.InNamespace(namespace)); err != nil { }
```